Public/Src/Engine/Scheduler/PipExecutor.cs

```diff
@@ -2164,14 +2262,107 @@ private static void ReportFileAccesses(ExecutionResult processExecutionResult, F
+                                                operationContext,
+                                                environment,
+                                                state,
+                                                cacheableProcess,
+                                                weakFingerprint,
+                                                weakAugmentingPathSet,
+                                                weakAugmentingPathSetHash);
+
+                            BoxRef<ProcessStrongFingerprintComputationData> strongFingerprintComputation = new ProcessStrongFingerprintComputationData(
+                                weakAugmentingPathSetHash,
+                                new List<StrongContentFingerprint>() { StrongContentFingerprint.AugmentedWeakFingerprintMarker },
+                                weakAugmentingPathSet);
+
+                            // Add the computation of the augmenting weak fingerprint
+                            strongFingerprintComputationList.Add(strongFingerprintComputation);
+
+                            if (observedInputProcessingResult.Status == ObservedInputProcessingStatus.Success)
+                            {
+                                // Add marker selector with weak augmenting path set
+                                var addAugmentationResult = await cache.TryPublishCacheEntryAsync(
+                                    cacheableProcess.Process,
+                                    weakFingerprint,
+                                    weakAugmentingPathSetHash,
+                                    StrongContentFingerprint.AugmentedWeakFingerprintMarker,
+                                    CacheEntry.FromArray((new[] { weakAugmentingPathSetHash }).ToReadOnlyArray(), "AugmentWeakFingerprint"));
+
+                                addAugmentingPathSetResultDescription = addAugmentationResult.Succeeded
+                                    ? addAugmentationResult.Result.Status.ToString()
+                                    : addAugmentationResult.Failure.Describe();
+
+                                augmentedWeakFingerprint = new WeakContentFingerprint(computedStrongFingerprint.Hash);
+
+                                strongFingerprintComputation.Value = strongFingerprintComputation.Value.ToSuccessfulResult(
+                                    computedStrongFingerprint,
+                                    observedInputProcessingResult.ObservedInputs);
+                            }
+                            else
+                            {
+                                addAugmentingPathSetResultDescription = observedInputProcessingResult.Status.ToString();
+                            }
+                        }
+                        else
+                        {
+                            addAugmentingPathSetResultDescription = weakAugmentingPathSetHashResult.Failure.Describe();
+                        }

```

Comment:
I don't like that errors are logged as verbose messages... I think we need to emit warning/errors to the log